Source/Message/MessageClient.swift

```diff
@@ -21,10 +21,22 @@
 import Foundation
 
 /// Message HTTP client.
-open class MessageClient: CompletionHandlerType<Message> {
+public class MessageClient {
+    
+    /// Alias for closure to handle a service response along with a Message object.
+    public typealias ObjectHandler = (ServiceResponse<Message>) -> Void
```

Comment:
I think it IS more clear that way personally. This was done as part of a refactor to get the typealias out of the base class, which existed solely for the purpose of creating and exposing generified typealiases. So I think we took some steps in the right direction here, but there's definitely more refactoring that could be done here.